test/dashboard/lesson_editor_test.exs

```diff
@@ -389,6 +389,29 @@ defmodule UneebeeWeb.DashboardLessonEditorLiveTest do
       assert updated_lesson.description == attrs.description
     end
 
+    test "edits a lesson information in realtime", %{conn: conn, course: course} do
+      lesson = lesson_fixture(%{course_id: course.id})
+      lesson_step_fixture(%{lesson_id: lesson.id, order: 1})
+      lesson_step_fixture(%{lesson_id: lesson.id, order: 2})
+
+      {:ok, lv, _html} = live(conn, ~p"/dashboard/c/#{course.slug}/l/#{lesson.id}/s/2")
+
+      {:ok, view, _html} =
+        lv
+        |> element(~s|a[href="/dashboard/c/#{course.slug}/l/#{lesson.id}/s/2/edit_step"]:fl-icontains("edit")|)
+        |> render_click()
+        |> follow_redirect(conn, edit_lesson_redirect_link(course, lesson))
+
+      assert_lesson_name(view)
+      assert_lesson_description(view)
+
+      attrs = %{name: "Updated lesson name", description: "Updated lesson description"}
+      view |> form(@lesson_form, lesson: attrs) |> render_submit()
+
+      assert render(view) =~ "Updated lesson name"
+      assert render(view) =~ "Updated lesson description"
```

Comment:
This can lead to a false positive in case the "Updated lesson name" text is present somewhere else in the page. For example, if you remove your changes you'll the first assertion (updated lesson name) actually passes, even though the feature is broken.

We can minimize this by using `has_element?/3` instead:

```suggestion
      assert has_element?(view, "h1", "Updated lesson name")
      assert has_element?(view, "p", "Updated lesson description")
```

Reply:
@wceolin Thank you for the update `has_element?/3` works better.